.github/ISSUE_TEMPLATE/Platform _Epic_Template.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
I think I would delete this whole table. All these things are already tracked by GitHub (Title, labels, etc).